src/SFA.DAS.AANHub.Application.UnitTests/Members/Commands/PostMemberStatus/PostMemberRemoveCommandValidatorTests.cs

```diff
@@ -0,0 +1,121 @@
+    [TestCase("ac44a17b-f843-4e1f-979b-aa95c0fe44f2", UserType.Apprentice, false, MemberIdValidator.MemberIdNotFoundErrorMessage)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Apprentice, true)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Employer, true)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Admin, false, MemberIdValidator.MemberIdNotFoundErrorMessage)]
+    public async Task ValidateMemberId(string memberId, UserType userType, bool isValid, string? errorMessage = null)
+    {
+        Mock<IMembersReadRepository> repositoryMock = new();
+        repositoryMock.Setup(r => r.GetMember(Guid.Parse("f5521677-7733-4416-b5a7-4c7a231fe469"))).ReturnsAsync(new Member() { Status = MembershipStatus.Live, UserType = userType });
+        PostMemberRemoveCommandValidator sut = new(repositoryMock.Object);
+        PostMemberRemoveCommand target = new();
+        if (memberId != null) target.MemberId = Guid.Parse(memberId);
+
+        var result = await sut.TestValidateAsync(target);
+
+        if (isValid)
+        {
+            result.ShouldNotHaveValidationErrorFor(s => s.MemberId);
+        }
+        else
+        {
+            if (userType == UserType.Admin)
+            {
+                result
+                .ShouldHaveValidationErrorFor(s => s.MemberId)
+                .WithErrorMessage(
+                    string.IsNullOrWhiteSpace(errorMessage)
+                    ? MemberIdValidator.MemberIdNotFoundErrorMessage
+                    : errorMessage);
+            }
+            else
+            {
+                result
+                .ShouldHaveValidationErrorFor(s => s.MemberId)
+                .WithErrorMessage(
+                    string.IsNullOrWhiteSpace(errorMessage)
+                    ? MemberIdValidator.MemberIdEmptyErrorMessage
+                    : errorMessage);
+            }
+        }
+    }
+
+    [TestCase(null, UserType.Admin, false, false, AdminMemberIdValidator.RequestedByMemberIdMustNotBeEmpty)]
+    [TestCase("00000000-0000-0000-0000-000000000000", UserType.Admin, false, false, AdminMemberIdValidator.RequestedByMemberIdMustNotBeEmpty)]
+    [TestCase("ac44a17b-f843-4e1f-979b-aa95c0fe44f2", UserType.Admin, false, false, AdminMemberIdValidator.RequestedByMemberIdMustBeAdmin)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Admin, false, true)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Apprentice, false, false, AdminMemberIdValidator.RequestedByMemberIdMustBeAdmin)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Employer, false, false, AdminMemberIdValidator.RequestedByMemberIdMustBeAdmin)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Admin, true, true)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Apprentice, true, true)]
+    [TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Employer, true, true)]
```

Comment:
I was going to suggest we add couple of cases here... 
```
[TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Apprentice, false, true)]
[TestCase("f5521677-7733-4416-b5a7-4c7a231fe469", UserType.Employer, false, true)]
```
But it seems that the status and regional chair checks have both been merged in one single validation.